doc/articles/toc.yml

```diff
@@ -429,6 +429,10 @@
         href: xref:Uno.HotDesign.GetStarted.Guide
       - name: Counter Tutorial
         href: xref:Uno.HotDesign.GetStarted.CounterTutorial
+      - name: Advanced
+        items:
+          - name: Canvas
+            href: xref:Uno.HotDesign.Canvas
```

Comment:
Adjust toc with the latest structure proposed. Same as in https://github.com/unoplatform/uno/pull/20175